grader/utils.py

```diff
@@ -2,7 +2,7 @@
 
 
 def get_url(endpoint):
-    return "http://{}:{}{}{}".format(API_HOSTNAME, API_PORT, API_PROXY, endpoint)
```

Comment:
can we make HTTPS vs HTTP a part of `config.py`?

Reply:
for local testing we required HTTP and in production we need HTTPS

Reply:
Oh sorry I just realized that. Fixed already.